casper/src/test/scala/coop/rchain/casper/batch2/FinalizerTest.scala

```diff
@@ -138,16 +130,12 @@ class FinalizerTest extends FlatSpec with Matchers with BlockGenerator with Bloc
                 dag,
                 faultToleranceThreshold = -1,
                 currLFBHeight = 0L,
-                m => (lfbStore = m).pure[Task],
-                m => finalisedStore.update(v => v ++ m)
+                m => (lfbStore = m).pure[Task] <* finalisedStore.update(v => v + m)
```

Comment:
Finalizer test is adjusted, because now finalizer only cares about single effect - when new LFB is found. How to find hashes that have to be finalized indirectly (and finalize them) - now is the task for BlockDagStorage.
```
def recordDirectlyFinalized(
      directlyFinalizedHash: BlockHash,
      finalizationEffect: Set[BlockHash] => F[Unit]
  ): F[Unit] =
```